asciidoctorj-core/src/test/java/org/asciidoctor/extension/GistMacro.java

```diff
@@ -19,7 +19,7 @@ public Block process(AbstractBlock parent, String target, Map<String, Object> at
        		"<script src=\"https://gist.github.com/"+target+".js\"></script>\n" + 
        		"</div>"; 
        
-       return createBlock(parent, "pass", Arrays.asList(content), attributes, this.getConfig());
```

Comment:
This only worked because the default config property :content_model => :attributes was not visible in the Java part.